hack/build.sh

```diff
@@ -38,10 +38,10 @@ else
   pushd "$TEMP/docs-main"; mkdocs build -f mkdocs.yml -d $SITE/development; popd
 
   # Latest release branch to /docs
-  git clone --depth 1 -b ${DOCS_BRANCHES[0]} https://github.com/${GIT_SLUG} "$TEMP/docs-$latest"
+  git clone --depth 1 -b ${RELEASE_BRANCHES[0]} https://github.com/${GIT_SLUG} "$TEMP/docs-$latest"
   curl -f -L --show-error https://raw.githubusercontent.com/knative/serving/${RELEASE_BRANCHES[0]}/docs/serving-api.md -s > "$TEMP/docs-$latest/docs/reference/api/serving-api.md"
   curl -f -L --show-error https://raw.githubusercontent.com/knative/eventing/${RELEASE_BRANCHES[0]}/docs/eventing-api.md -s > "$TEMP/docs-$latest/docs/reference/api/eventing-api.md"
-  pushd "$TEMP/docs-$latest"; KNATIVE_VERSION=${RELEASE_BRANCHES[0]} SAMPLES_BRANCH="${DOCS_BRANCHES[0]}" mkdocs build -d $SITE/docs; popd
+  pushd "$TEMP/docs-$latest"; KNATIVE_VERSION=${RELEASE_BRANCHES[0]} SAMPLES_BRANCH="${RELEASE_BRANCHES[0]}" mkdocs build -d $SITE/docs; popd
```

Comment:
```suggestion
  pushd "$TEMP/docs-$latest"; KNATIVE_VERSION=${RELEASE_BRANCHES[0]//release-} SAMPLES_BRANCH="${RELEASE_BRANCHES[0]}" mkdocs build -d $SITE/docs; popd
```
for consistency

Reply:
I think macros.py also needs to be adjusted

```
        else:
            if version.startswith("1."):
                return 'https://github.com/{org}/{repo}/releases/download/knative-v{version}/{file}'.format(
                    repo=repo,
                    file=file,
                    version=version,
                    org=org)
            else:
                return 'https://github.com/{org}/{repo}/releases/download/v{version}/{file}'.format(
                    repo=repo,
                    file=file,
                    version=version,
                    org=org)
```
https://github.com/knative/docs/blob/main/hack/macros.py#L28
You can do a full site build locally and test it in your computer with
```
./hack/docker/test.sh
```

Reply:
just realized that the macro.py needs to be updated in the docs branches also `release-0.26` `release-1.0` `release-1.1` and `release-1.2` so it will take the change 5 PRs, main + 4

Reply:
let me know if you need help with the PRs, I will assist with the merge of all of them at the same time

Reply:
oddly I did the site build and it didn't fail